IV9VBS Commit (Extra colors)

NaominoDino opened the pull request:
Adding More Colors

NaominoDino committed:
IV9VBS Commit (Extra colors)

Adding More Colors

NaominoDino committed:
Multiple Browser Support

Now Opens in Mozilla, Chrome, Safari

NaominoDino committed:
Help

Help Button now opening + Working!

csurgay reviewed (changes requested):
Inkább ezt kérném bele, valami sárgaféle kell, ami látszik is, az OrangeRed viszont nem üt el eléggé a pirostól. Köszi

const COLORS=["black","red","green","blue","#dddd00","purple", "brown", "Chartreuse", "DeepPink"];

NaominoDino committed:
Update graphics.js

Color changing

NaominoDino commented:
> Inkább ezt kérném bele, valami sárgaféle kell, ami látszik is, az OrangeRed viszont nem üt el eléggé a pirostól. Köszi
> 
> const COLORS=["black","red","green","blue","#dddd00","purple", "brown", "Chartreuse", "DeepPink"];

Javítottam

NaominoDino committed:
Merge branch 'master' into master

NaominoDino committed:
Detecting Device

Detecting Phone, Tablet, PC/Notebook. => Showing + Add button for Mobile Platform (Not implemented backend)
